### ISSUE TITLE
IDE1016 should not warn when name is all underscores

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn#40713]
```cs
void M()
{
    //                            ↓ IDE1006should not warn here
    Console.CancelKeyPress += (_, __) => { };
}
```

**Comment by mavasani** [comment]
Design meeting review:
We will accept a PR to implement the requested behavior.
